packages/block-editor/src/components/spacing-sizes-control/spacing-input-control.js

```diff
@@ -228,6 +228,9 @@ export default function SpacingInputControl( {
 						value={ customRangeValue }
 						min={ 0 }
 						max={ 300 }
```

Comment:
Addressing thje comment by @talldan above, we should probably change the `max` value depending on the `selectedUnit`. This sets max to 300 if using `px`, and 10 for other units.
```suggestion
						max={
							'px' === selectedUnit ? 300 : 10
						}
```